ui-tests/src/test/java/org/openmrs/reference/MergeVisitsTest.java

```diff
@@ -36,13 +35,14 @@ public void setUp() {
     }
 
     @Test
-    @Ignore
     @Category(BuildTests.class)
     public void mergeVisitsTest() {
         FindPatientPage findPatientPage = homePage.goToFindPatientRecord();
         findPatientPage.enterPatient(patient.identifier);
+        findPatientPage.waitForPageToLoad();
         ClinicianFacingPatientDashboardPage clinicianFacingPatientDashboardPage = findPatientPage.clickOnFirstPatient();
         PatientVisitsDashboardPage patientVisitsDashboardPage = clinicianFacingPatientDashboardPage.goToRecentVisits();
+        patientVisitsDashboardPage.waitForPageToLoad();
```

Comment:
`Waiting the page to load` ensures that the `classLoader` avails the page for the next action to is effective. Otherwise the next action `clickOnActions()` performs the operation on the [element](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/b58e4d732d5fc2eca1cd1aafc14a11839ba43863/ui-tests/src/test/java/org/openmrs/reference/page/PatientVisitsDashboardPage.java#L27) when the page that contains the element in `context` is not yet loaded